src/integrationTest/resources/application.yml

```diff
@@ -5,11 +5,41 @@ spring:
     platform: h2
     username: ${POSTGRES_USERNAME:dbuserprofile}
     password: ${POSTGRES_PASSWORD:dbuserprofile}
-
+  main:
+    allow-bean-definition-overriding: true
+    security:
+      oauth2:
+        client:
+          provider:
+            oidc:
+              issuer-uri: ${OPEN_ID_API_BASE_URI:https://idam-web-public.aat.platform.hmcts.net/o}
```

Comment:
will this call actual AAT SIDAM for issuer url  for integration test cases? Ideally mocks should be used just like used in PRD.

Reply:
for user profile not required mock since we are not doing any role based validation with bearer token